docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -4940,10 +4940,6 @@ Model.distinct.pluck(:id)
 | EnforcedStyle
```

Comment:
Side observation.

Examples are outdated, too.
```ruby
# this will ~return a Relation~ that pluck is called on
Model.where(cond: true).pluck(:id).uniq
```
check:
```ruby
> x=User.where('1 = 1').pluck(:id);
SELECT "users"."id" FROM "users" WHERE (1 = 1)
> x.class
=> Array
```
`uniq` will be called on an array.

```ruby
# an association on an instance will ~return a CollectionProxy~
instance.assoc.pluck(:id).uniq
```
It will return an `Array`.

```ruby
# good
Model.distinct.pluck(:id)
```
I understand the reasoning, but the usage of `id` breaks it.
```ruby
Model.distinct.pluck(:favourite_food)
```
would make more sense.
I believe the usage of `id` in those examples is more confusing than helpful.

Reply:
Thanks for the feedback. That point can open a PR separately. Can you open a PR if you like?